packages-common/pluggable-widgets-tools/utils/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from "./native/common";
```

Comment:
as discussed, let's rename the folder to `src`

Reply:
Renamed it.